exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,115 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. But how to assess 
+the level of complexity of a given text, and how to teach the computer to do so? 
+In this exercise, you will write a program determining how difficult a text is 
+and for what age it is best suited.
+
+For a formula for calculating the complexity, we will use 
+[Automated readability index (ARI)][wiki] 
+developed in 1967 year. The ARI assesses the U.S. grade level required to read 
+a piece of text. In some ways, it is similar to other formulas. Its difference 
+is rather than counting syllables, it counts characters. The more characters, 
+the harder the word. It also counts sentences. This sets it apart from some 
+other formulas.
+
+## Text Preview
+
+Your program should display the first three sentences of the text. If the sentence 
+is longer than 50 characters, truncate the sentence and add an ellipsis at the end. 
+Put an ellipsis on the fourth line if the text is more than three sentences.
```

Comment:
```suggestion
Your program should display the first three sentences of the text, but no more than 50 characters. If the first three sentences exceed 50 characters, truncate them and add an ellipsis at the end. Otherwise, if the text is more than three sentences, put an ellipsis on the fourth line.
```